samples/Tests/Column.Explicit.Size.json

```diff
@@ -0,0 +1,52 @@
+{
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+		    "type": "ColumnSet",
+		    "columns": [
+		        {
+		            "width": "50px",
+		            "items": [
+		                {
+		                    "type": "Image",
```

Comment:
>		                     [](start = 0, length = 22)

Make the spacing consistent. You should use tabs in the json samples.

Reply:
o.k.


---
In reply to: [186202407](https://github.com/Microsoft/AdaptiveCards/pull/1427#discussion_r186202407) [](ancestors = 186202407)